internal/git/git.go

```diff
@@ -472,6 +472,25 @@ func (g *Git) WritePRBody(prNumber *int, body string) error {
 	return nil
 }
 
+func (g *Git) WritePRComment(prNumber *int, fileName, body string, line int) error {
```

Comment:
nit: does prNumber need to be a pointer? Is it an option to leave it as nil?

Reply:
Also, should this method be in the github client? `github.go`? Seems like more of a github thing than a git thing

Reply:
Changed signature to `int`.

Regarding github vs. git, we don't have that separation elsewhere in the action. All the logic for creating/updating gen PRs (which uses `go-git`) is muddled with logic that gets the latest tag or download link (which uses `go-github`). The `Git` struct in `git.go` has clients for both of these. Can separate it out but would rather refactor it in a different PR.

Reply:
Gotcha 👍